Gemfile

```diff
@@ -76,7 +76,7 @@ gem "redcarpet", "~> 3.5" # A fast, safe and extensible Markdown to (X)HTML pars
 gem "reverse_markdown", "~> 1.3" # Map simple html back into markdown
 gem "rolify", "~> 5.2" # Very simple Roles library
 gem "rouge", "~> 3.10" # A pure-ruby code highlighter
-gem "rubyzip", "~> 1.2" # Rubyzip is a ruby library for reading and writing zip files
+gem "rubyzip", "~> 1.2", ">= 1.3.0" # Rubyzip is a ruby library for reading and writing zip files
```

Comment:
Why not just `gem "rubyzip",  ">= 1.3.0"`  ? :thinking:

Reply:
Honestly, no particular reason, the algorithm works the same I think, see the line with Rails and other gems with the same double rule